conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
@vavuthu shouldn't it be vsan?

Reply:
nope, its for vmfs. we already have for vsan